external/textprogressbar/textprogressbar.m

```diff
@@ -17,16 +17,18 @@ function textprogressbar(c)
 %
 %      - renamed strCR to WAITBAR_HANDLE
 %      - changed persistent type of strCR to global type
+%      - Changes To the progressbar text to be consistent (to allow
+%        intermittent output)
 
 %% Initialization
 global WAITBAR_HANDLE;           %   Carriage return pesistent variable
-
+persistent WAITBAR_PROGRESS;
```

Comment:
I am just wondering if `persistent` is enough, or if we should have `global` ...